README.md

```diff
@@ -1,20 +1,24 @@
 # Rancher Product Docs
 
-Welcome to the Rancher Product docs. We are currently a work in progress. 
+Welcome to the Rancher Product docs.
 
-🚧🛠️👷🛠️🚧
+This repo is used to migrate content from the [Rancher docs](https://ranchermanager.docs.rancher.com/) site to publish it on the [SUSE Documentation site](https://documentation.suse.com/). It runs on the [Antora](https://antora.org/) framework.
 
-We are using this repo to migrate the current [Rancher docs](https://ranchermanager.docs.rancher.com/) site to run on the [Antora](https://antora.org/) framework. 
+See the [Rancher software](https://github.com/rancher/rancher) repo if you have questions or requests for the Rancher platform.

```

Comment:
```suggestion
For questions or feature requests about Rancher software, refer to [Rancher GitHub repository](https://github.com/rancher/rancher).
```

Reply:
Just a small change,
IMO: "see" might not be the most inclusive term for visually impaired users. Consider using "refer to" instead.